src/libraries/System.Security.Cryptography.Cng/tests/RSACngPkcs8TestsPssSaltLength.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public abstract class RSACngPkcs8TestsPssSaltLength : CngPkcs8Tests<RSACng>
```

Comment:
I don't think we need a whole hierarchy around this concept.  The CngPkcs8Tests VerifyMatch is more about just doing a sanity check than testing a key to the limits.

Unless CNG's PKCS#8 import/export supports salt length restrictions that we're trying to test aren't getting ignored, all of the new stuff deriving from CngPkcs8Tests can be deleted.